scripts/email/templates/dryrun.py

```diff
@@ -0,0 +1,5 @@
+BCC = ["hyz4@cornell.edu"]
+SUBJECT = "Sign up for CoursePlan 🔫"
+HTML = (
+    "<strong>This is a requirement for your continued presence on CoursePlan.</strong>"
```

Comment:
perfect that this is just straight HTML! shouldn't be terrible to implement the frontend then :) looks good!